src/main/java/ru/itmo/sd/deadliner2bot/model/DailyNotification.java

```diff
@@ -0,0 +1,22 @@
+package ru.itmo.sd.deadliner2bot.model;
+
+import javax.persistence.*;
+import java.math.BigInteger;
+import java.util.Date;
+
+@Entity
+@Table(name = "daily_notifications")
+public class DailyNotification {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "daily_notification_id")
+    private BigInteger dailyNotificationId;
+
+    @Column(name = "notification_time")
+    private Date notificationTime;
```

Comment:
Are you sure about `Date`? It seems to be [deprecated](https://docs.oracle.com/javase/8/docs/api/java/util/Date.html) in favor to [`Calendar`](https://docs.oracle.com/javase/8/docs/api/java/util/Calendar.html).
However, I am unsure that JPA can map `TIMESTAMP` of PostgreSQL into `Calendar`.

Reply:
As I see, both `Date` and `Calendar` are deprecated since Java 8. From Java 8 `LocalDateTime` is more convenient to map `timestamp`

Reply:
Nice, let's use `LocalDateTime` then.
https://www.baeldung.com/jpa-java-time#after-java-8